src/transformers/models/openai/modeling_openai.py

```diff
@@ -130,7 +130,7 @@ def load_tf_weights_in_openai_gpt(model, config, openai_checkpoint_folder_path):
     return model
 
 
-ACT_FNS = {"relu": nn.ReLU, "silu": silu, "gelu": gelu_new, "swish": silu}
+ACT_FNS = {"relu": torch.relu, "silu": silu, "gelu": gelu_new, "swish": silu}
```

Comment:
```suggestion
ACT_FNS = {"relu": nn.ReLU(), "silu": silu, "gelu": gelu_new, "swish": silu}
```
both would work. Since `silu` `gelu` and `swish` are all `nn.Modules` let's go with that